src/components/map/Map.js

```diff
@@ -27,11 +30,21 @@ const layerType = {
     external: ExternalLayer,
 };
 
+const styles = {
```

Comment:
Since we're introducing `styles` here can we remove the inline styles from this file at the same time instead of keeping both?